movement_primitives/dmp/_dmp.py

```diff
@@ -296,7 +296,7 @@ def dmp_step_euler(
     pass
 
 
-class DMP(DMPBase):
+class DMP(WeightParametersMixin, DMPBase):
```

Comment:
Is there a reason for this order?

Reply:
inspired by [scikit](https://github.com/scikit-learn/scikit-learn/blob/bbd64c16f403f6b1b982c95673324db84b1196e1/sklearn/gaussian_process/_gpr.py#L24)

Reply:
or as [they](https://realpython.com/lessons/multiple-inheritance-python/) say, "A mixin is a class that doesn’t care about its position in the hierarchy, but just provides one or more convenience methods"

Reply:
OK, thanks